src/ai/backend/agent/agent.py

```diff
@@ -1839,11 +1845,24 @@ async def create_kernel(
                 kernel_config["image"]["digest"],
                 AutoPullBehavior(kernel_config.get("auto_pull", "digest")),
             )
+            timeout = cast(
```

Comment:
To properly illustrate purpose of this timeout value please rename the variable.
```suggestion
            image_pull_timeout = cast(
```